.github/workflows/build_and_test.yml

```diff
@@ -435,8 +435,7 @@ jobs:
             ./vendor/postgres/Dockerfile
 
   neon-image:
-    # force building for all 3 images
-    if: needs.dockerfile-check.outputs.value == 'true'
+    #if: needs.dockerfile-check.outputs.value == 'true'
```

Comment:
Let's delete the whole `dockerfile-check` job if we don't use it

Reply:
Done